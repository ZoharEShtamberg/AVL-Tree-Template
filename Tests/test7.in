add_country 1 0
add_team 11 1 1
add_contestant 1 1 1 800
add_contestant 2 1 1 700
add_contestant 3 1 1 600
add_contestant 4 1 1 500
add_contestant 10 1 1 80
add_contestant 20 1 1 70
add_contestant 30 1 1 60
add_contestant 40 1 1 50
add_contestant 100 1 1 8
add_contestant 200 1 1 7
add_contestant 300 1 1 6
add_contestant 400 1 1 5
add_contestant_to_team 11 1
add_contestant_to_team 11 2
add_contestant_to_team 11 3
add_contestant_to_team 11 4
add_contestant_to_team 11 10
add_contestant_to_team 11 20
add_contestant_to_team 11 30
add_contestant_to_team 11 40
add_contestant_to_team 11 100
add_contestant_to_team 11 200
add_contestant_to_team 11 300
add_contestant_to_team 11 400
add_team 22 1 1
add_contestant 5 1 1 400
add_contestant 6 1 1 300
add_contestant 7 1 1 200
add_contestant 50 1 1 40
add_contestant 60 1 1 30
add_contestant 70 1 1 20
add_contestant 500 1 1 4
add_contestant 600 1 1 3
add_contestant 700 1 1 2
add_contestant_to_team 22 5
add_contestant_to_team 22 6
add_contestant_to_team 22 7
add_contestant_to_team 22 50
add_contestant_to_team 22 60
add_contestant_to_team 22 70
add_contestant_to_team 22 500
add_contestant_to_team 22 600
add_contestant_to_team 22 700
austerity_measures 22
unite_teams 11 22
add_contestant_to_team 22 5
get_team_strength 11
remove_contestant_from_team 11 1
remove_contestant_from_team 11 2
remove_contestant_from_team 11 3
remove_contestant_from_team 11 4
remove_contestant_from_team 11 5
remove_contestant_from_team 11 6
add_team 33 1 1
add_contestant_to_team 33 1
add_contestant_to_team 33 2
add_contestant_to_team 33 3
add_contestant_to_team 33 4
add_contestant_to_team 33 5
add_contestant_to_team 33 6
unite_teams 11 33
update_contestant_strength 1 -791
get_team_strength 11
austerity_measures 11
