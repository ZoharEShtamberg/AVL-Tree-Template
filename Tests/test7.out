add_country: SUCCESS
add_team: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_team: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
austerity_measures: SUCCESS, 630
unite_teams: SUCCESS
add_contestant_to_team: FAILURE
get_team_strength: SUCCESS, 888
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
add_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
unite_teams: SUCCESS
update_contestant_strength: SUCCESS
get_team_strength: SUCCESS, 788
austerity_measures: SUCCESS, 930
